test/DeploymentHelper.test.ts

```diff
@@ -0,0 +1,222 @@
+      );
+      await expect(deploymentHelper.deploy()).to.be.not.reverted;
+    });
+  });
+
+  describe("deployed CowProtocolVirtualToken", async function () {
+    beforeEach(async function () {
+      await multiTokenMediatorHome.mock.bridgedTokenAddress.returns(
+        bridgedCowTokenAddress,
+      );
+      deploymentHelper = await DeploymentHelper.deploy(
+        ...constructorInput(ContractName.DeploymentHelper, deploymentParams),
+      );
+      const vCowAddress = await deploymentHelper.callStatic.deploy();
+      await deploymentHelper.deploy();
+      cowProtocolVirtualToken = await ethers.getContractAt(
+        "CowProtocolVirtualToken",
+        vCowAddress,
+      );
+    });
+
+    it("has expected name", async () => {
+      expect(await cowProtocolVirtualToken.name()).to.equal(
+        metadata.virtual.name,
+      );
+    });
+
+    it("has expected symbol", async () => {
+      expect(await cowProtocolVirtualToken.symbol()).to.equal(
+        metadata.virtual.symbol,
+      );
+    });
+
+    it("has 18 decimals", async () => {
+      expect(await cowProtocolVirtualToken.decimals()).to.equal(
+        metadata.virtual.decimals,
+      );
+    });
+
+    it("has expected merkle root", async () => {
+      expect(await cowProtocolVirtualToken.merkleRoot()).to.equal(merkleRoot);
+    });
+
+    it("has expected real deploymentHelper", async () => {
+      expect(await cowProtocolVirtualToken.cowToken()).to.equal(
+        bridgedCowTokenAddress,
+      );
+    });
+
+    it("sets the expected community funds target", async function () {
+      expect(await cowProtocolVirtualToken.communityFundsTarget()).to.equal(
+        communityFundsTarget,
+      );
+    });
+
+    it("sets the expected USDC deploymentHelper", async function () {
+      expect(await cowProtocolVirtualToken.usdcToken()).to.equal(
+        constants.AddressZero,
+      );
+    });
+
+    it("sets the expected USDC price", async function () {
+      expect(await cowProtocolVirtualToken.usdcPrice()).to.equal(0);
+    });
+
+    it("sets the expected GNO deploymentHelper", async function () {
+      expect(await cowProtocolVirtualToken.gnoToken()).to.equal(
+        gnoToken.address,
+      );
+    });
+
+    it("sets the expected GNO price", async function () {
+      expect(await cowProtocolVirtualToken.gnoPrice()).to.equal(gnoPrice);
+    });
+
+    it("sets the expected wrapped native deploymentHelper", async function () {
+      expect(await cowProtocolVirtualToken.wrappedNativeToken()).to.equal(
+        wrappedNativeToken.address,
+      );
+    });
+
+    it("sets the expected native deploymentHelper price", async function () {
+      expect(await cowProtocolVirtualToken.nativeTokenPrice()).to.equal(
+        nativeTokenPrice,
+      );
+    });
+
+    it("sets the expected team controller", async function () {
+      expect(await cowProtocolVirtualToken.teamController()).to.equal(
+        constants.AddressZero,
+      );
+    });
```

Comment:
I agree that this format is nice for consistency with the other tests as well as having an explicit test for each parameter.
However, since it's very easy to forget a deployment parameter, maybe this could rather be tested using `getDeployArgsFromVirtualToken`, similar to [this](https://github.com/gnosis/cow-token/blob/main/test/proposal.test.ts#L222-L235)?